unattended.sh

```diff
@@ -496,16 +495,9 @@ function install_centreon_repo() {
 
 	log "INFO" "Centreon official repositories installation..."
 	$PKG_MGR -q clean all
-
-	rpm -q centreon-release-$CENTREON_MAJOR_VERSION >/dev/null 2>&1
-	if [ $? -ne 0 ]; then
-		$PKG_MGR -q install -y $RELEASE_RPM_URL
-		if [ $? -ne 0 ]; then
-			error_and_exit "Could not install Centreon repository"
-		fi
-	else
-		log "INFO" "Centreon repository seems to be already installed"
-	fi
+	yum install -y yum-utils
+	yum-config-manager --add-repo \
+		"https://download.centreon.com/static/centreon-${CENTREON_MAJOR_VERSION}-${EL_MAJOR_VERSION}.repo"
```

Comment:
@Nohzoh we may include the repo files directly in the git repo ?

Reply:
We can.
The only limitation I see is that the install will rely on Github availability.
It is not much of a risk.

Reply:
I think if that happens, the script itself will not be available